src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -1214,6 +1244,10 @@ private static InstructionSetFlags ExpandInstructionSetByReverseImplicationHelpe
                         resultflags.AddInstructionSet(InstructionSet.X86_AVX10v1_V512);
                     if (resultflags.HasInstructionSet(InstructionSet.X86_AVX512VBMI_VL))
                         resultflags.AddInstructionSet(InstructionSet.X86_AVX10v1_V512);
+                    if (resultflags.HasInstructionSet(InstructionSet.X86_AVX10v1))
+                        resultflags.AddInstructionSet(InstructionSet.X86_AVX10v2);
+                    if (resultflags.HasInstructionSet(InstructionSet.X86_AVX10v1_V512))
+                        resultflags.AddInstructionSet(InstructionSet.X86_AVX10v2_V512);
```

Comment:
This seems odd: it says that if AVX10v1 is supported, then AVX10v2 is supported. Is that correct? I assume the OPPOSITE is true.

Reply:
This is part of `ExpandInstructionSetByReverseImplicationHelper`, which exists to allow NAOT to enable any dependent ISAs if the user just specifies `avx10v2` on the command line.

Reply:
The opposite is true. @khushal1996 please correct this.

Reply:
@anthonycanino this part is correct (and is notably autogenerated by the tool). Its part of `ExpandInstructionSetByReverseImplicationHelper`